tbx/static_src/sass/components/_logo.scss

```diff
@@ -1,25 +1,49 @@
 @use 'config' as *;
 
 .logo {
-    width: 108px;
-    height: 20px;
     color: var(--color--heading);
 
-    @include media-query(menu) {
-        width: 158px;
-        height: 28px;
-    }
-
     @include high-contrast-light-mode() {
         color: var(--color--decoration);
     }
 
-    &__label {
-        @include font-size(size-four);
-        color: var(--color--theme-primary);
+    &--torchbox {
+        width: 112px;
+        height: 30px;
+
+        @include media-query(menu) {
+            width: 147px;
+            height: 38.964px;
+        }
+    }
+
+    &--logo-charity {
+        width: 189px;
+        height: 30px;
+
+        @include media-query(menu) {
+            width: 251px;
+            height: 38.97px;
+        }
+    }
+
+    &--logo-public {
+        width: 174px;
+        height: 30px;
+
+        @include media-query(menu) {
+            width: 227px;
+            height: 39.044px;
+        }
+    }
+
+    &--logo-wagtail {
```

Comment:
any chance we could decrease the logo or adjust the design so that it's not getting broken on smaller mobile phones like Galaxy S8+/iPhone SE? (see the header that's not 100% width and a logo a bit too close to the menu button)
![image](https://github.com/user-attachments/assets/9fc35b7b-5893-49ef-b112-e13927e75858)